aten/src/ATen/native/mkldnn/MKLDNNConversions.cpp

```diff
@@ -256,9 +257,41 @@ static Tensor mkldnn_reorder_conv_weight(
   }
 }
 
-static Tensor mkldnn_reorder_linear_weight(
+Tensor mkldnn_reorder_linear_weight(
     const Tensor& self,
     std::optional<int64_t> batch_size_opt) {
+
+  ideep::tensor result;
+#if defined(__aarch64__) && AT_MKLDNN_ACL_ENABLED()
```

Comment:
can you add a note here explaining why you cannot reuse existing implementation but have to create a special path for aarch64? We need a TODO to consolidate the implementation in the future.